src/opera/util/geo_utils.py

```diff
@@ -196,3 +214,37 @@ def get_geographic_boundaries_from_mgrs_tile(mgrs_tile_name):
                 lon_max = lon
 
     return lat_min, lat_max, lon_min, lon_max
+
+
+def get_gml_polygon_from_frame(frame_id, frame_geometries):
+    """
+    Returns the GML formatted polygon string for a single frame.
+
+    Parameters
+    ----------
+    frame_id : int
+        The ID of the frame to get the bounding box for.
+    frame_geometries : str
+        The path to the geojson containing frame geometries.
+
+    Returns
+    -------
+    bounding_polygon_gml_str : str
+        GML formatted bounding polygon string
+        
+    """
+    gdf_frames = get_frame_geodataframe(
+        frame_ids=[frame_id],
+        json_file=frame_geometries
+    )
+    
+    polygon = gdf_frames.loc[frame_id].geometry
+    
+    # Handles the case where polygon crosses the anti-meridian
+    # Translate the polygon with negative longitudes to be right of +180
+    if isinstance(polygon, MultiPolygon):
+        gg_neg, gg_pos = polygon.geoms
+        polygon = union_all([gg_pos, affinity.translate(gg_neg, 360)])
```

Comment:
This might be overkill for what I was looking for. Are we certain we need it? 

Without it it seem like we wouldn't need the following imports
```
from shapely import union_all, affinity
from shapely.geometry import MultiPolygon
```
So we wouldn't have to mock them, and then we wouldn't run afoul of the `isinstance` issue flagged by Riley.

`parse_bounding_polygon_from_wkt` should do the "right thing" for both types of polygons, so I'm fine with just passing through whatever we get from `get_frame_geodataframe`  without checking to see if its `MultiPolygon` or not.